trident/absorption_spectrum/absorption_line.py

```diff
@@ -213,11 +231,101 @@ def tau_profile(lambda_0, f_value, gamma, v_doppler, column_density,
     # tau_0
     tau_X = tau_factor * column_density * f_value / v_doppler
     tau0 = tau_X * lambda_0 * 1e-8
-
+    
```

Comment:
I think whitespace changes like these will fail the flake tests.  In general, try to leave whitespace alone.